### PR TITLE
depend on tar v2

### PR DESCRIPTION
**Pull request by isaacs**

### CODE REVIEW

**Comment by isaacs** [comment]
It's worth migrating to v3, though.  It's faster and better.

**Comment by Samurais** [comment]
+1

**Comment by moos** [comment]
Thanks for the heads-up and PR.  This is fixed by #12.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "url": "git://github.com/moos/wordnet-db.git"
   },
   "dependencies": {
-    "tar": "latest"
```
Comment (peterquek1983) [inline package.json]: J

**Pull request closed without merging**